blocks/priority-list/priority-list.js

```diff
@@ -98,12 +98,14 @@ export default async function decorate(block) {
   rightColumnContainer.append(textContainer);
   rightColumnContainer.append(cardsContainer);
   const imageContainer = div({ class: 'image-container' });
-  const imgElement = img({ src: cards.at(0).querySelector('img').src });
-  imageContainer.append(imgElement);
 
-  imageContainer.style.backgroundImage = `url(${
-    cards.at(0).querySelector('img').src
-  })`;
+  if (cards.at(0).querySelector('img')) {
```

Comment:
instead of querying multiple times store first query value in a variable and then use it

const firstCardImg = cards.at(0)?.querySelector('img'); // Ensure first card exists
if (firstCardImg) {
  const imgElement = img({ src: firstCardImg.src });
  imageContainer.append(imgElement);
  imageContainer.style.backgroundImage = `url(${firstCardImg.src})`;
}